VeterinaryAlina/AnimalFactory.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace VeterinaryAlina
+{
+    public enum AnimalType 
+    {
+        Cat, Dog, Fish, Hamster, Parrot
+    }
+    static class AnimalFactory
+    {
+        public static HomeAnimal CreateAnimal(AnimalType type, string name, DateTime birthYear, string genderOfAnimal, bool isHealthy,
+            bool isVaccinated, bool isAlergicToMedication, DateTime registrationDate, string animalBrid, string disease)
+        {
+            switch (type)
+            {
+                case AnimalType.Cat: return new HomeCat(name, birthYear, genderOfAnimal, isHealthy, isVaccinated,
```

Comment:
1) Multiple returns from the metod aren't a good practice
2) put "case ...:" on different line
3) use break